ReportSimulationOutput/measure.rb

```diff
@@ -227,10 +227,10 @@ def energyPlusOutputRequests(runner, user_arguments)
       include_timeseries_zone_temperatures = include_timeseries_zone_temperatures.is_initialized ? include_timeseries_zone_temperatures.get : false
       include_timeseries_airflows = include_timeseries_airflows.is_initialized ? include_timeseries_airflows.get : false
       include_timeseries_weather = include_timeseries_weather.is_initialized ? include_timeseries_weather.get : false
-      @user_output_variables = @user_output_variables.is_initialized ? @user_output_variables.get : false
+      user_output_variables = user_output_variables.is_initialized ? user_output_variables.get : nil
```

Comment:
No need for a global variable, it can easily be made a local variable and passed to `setup_outputs`. Also, it's not boolean so it shouldn't default to false if not provided.